src/Tribe/Assets.php

```diff
@@ -320,7 +346,7 @@ public static function maybe_get_min_file( $url ) {
 		// Check for all Urls added to the array.
 		foreach ( $urls as $partial_path ) {
 			$file_path = wp_normalize_path( $base_dir . $partial_path );
```

Comment:
This was already normalized above for $base_dir >> $base_dirs, it does not look needed anymore?

Reply:
Since we still need to normalize the `$partial_path` here it makes sense to not modify to prevent bugs, i dont know how it would react to having only the partial.

Reply:
Oh I see now the $url gets used here and that's the dynamic part, gotcha